tests/integration_tests/functional/test_api.py

```diff
@@ -1236,7 +1234,7 @@ def test_get_full_config_after_restoring_snapshot(bin_cloner_path):
     setup_cfg["metrics"] = None
     setup_cfg["mmds-config"] = {
         "version": "V1",
-        "network_interfaces": [DEFAULT_DEV_NAME],
+        "network_interfaces": [net_iface.dev_name],
```

Comment:
So, this one does not actually changes the names of the devices being used. It just uses the `NetIfaceConfig` object fields instead of the `DEFAULT_*_NAME` counterparts, even though the `NetIfaceConfig` objects actually use the defaults.

I guess this will be useful later on (?) If that is the case, could you add that information in the commit message?

Reply:
(here's the correct place)

Reply:
Hmm I don't remember but I thought this fixed something. Or maybe the issue got fixed with a later fix.

Will add to the commit body.